diff --git a/functions.php b/functions.php
index bfc6363..edbe8b0 100644
--- a/functions.php
+++ b/functions.php
@@ -105,7 +105,7 @@ class Theme {
        }
 
        function specify_theme_features() {
-               $before_widget = '<div id="%1$s" class="widget %2$s enthusiast-theme">';
+               $before_widget = '<div id="%1$s" class="widget %2$s enthusiast-theme widget-area">';
                $after_widget = '</div>';
                $before_title = '<h4 class="widget-title">';
                $after_title = '</h4>';
diff --git a/style.css b/style.css
index fd5bd96..d8a5a8e 100644
--- a/style.css
+++ b/style.css
@@ -592,7 +592,6 @@ li li a .sf-sub-indicator {
 .home-middle-full-width {
        clear:both;
        width:960px;
-       margin:10px 0px;
        float:left;
 }
 
@@ -667,7 +666,7 @@ div.menu-left-container ul a:hover {
 ------------------------------------------------------------ */
 
 #inner {
-       margin: 20px auto;
+       margin: 0px auto;
        overflow: hidden;
        width: 960px;
 }
